keyboards/gmmk/pro/pro.c

```diff
@@ -14,3 +14,15 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 #include "pro.h"
+
+#ifdef RGB_MATRIX_ENABLE
```

Comment:
@drashna without this the GMMK Pro wouldn't wake up from a USB suspend. I guess this now counts as a bugfix 😛

Reply:
With it, it's somewhat random whether or not it will wake, but I won't spend too much time on this. I can revert this bit if you think it's better to leave this out.